### PR TITLE
Adding expanduser to replace-certificates

### PR DESCRIPTION
**Pull request by YoniItzhak**

### CODE REVIEW

File: cloudify_cli/replace_certificates_config.py [hunk 1]
```diff
@@ -162,18 +163,19 @@ def replace_certificates(self):
...
+        self.logger.info('Successfully replaced certificates')
 
     def _handle_new_ca_certs(self):
         """Replace the CLI and client CA certs"""
         if env.profile.rest_certificate:
             new_manager_ca = self.config_dict['manager'].get('new_ca_cert')
             if new_manager_ca:
-                env.profile.rest_certificate = new_manager_ca
+                expanded_path = expanduser(new_manager_ca)
+                env.profile.rest_certificate = expanduser(expanded_path)
```
Comment (tehasdf) [inline cloudify_cli/replace_certificates_config.py:176]: no need to expand it twice :P

Comment (YoniItzhak) [reply cloudify_cli/replace_certificates_config.py]: Thanks! Forgot to delete it after creating the variable

File: cloudify_cli/replace_certificates_config.py [hunk 2]
```diff
@@ -162,18 +163,19 @@ def replace_certificates(self):
         self._handle_new_ca_certs()
         # Passing only the new CA cert to the agents
         self._pass_new_ca_certs_to_agents(bundle=False)
-        self.logger.info('Validating status is healthy')
+        self.logger.info('\nValidating status is healthy')
```
Comment (geokala) [inline cloudify_cli/replace_certificates_config.py:166]: Why the newline on a logger line?

Comment (YoniItzhak) [reply cloudify_cli/replace_certificates_config.py:166]: Just to separate it from the `Replacing certificates on host <host-ip>` logs. As a best-practice, logs shouldn't include newlines.

**Review by tehasdf — APPROVED** [review]

**Review by geokala — APPROVED** [review]

**Pull request merged**